### PR TITLE
Clarify reserved parameters, fix minor inconsistencies

### PR DESCRIPTION
**Pull request by Siegrift**
https://api3dao.atlassian.net/browse/AN-616

### CODE REVIEW

File: docs/ois/v1.0.0/ois.md [hunk 1]
```diff
@@ -457,11 +458,13 @@ Allowed values: `_type`, `_path` or `_times`
 #### 5.4.2. `fixed`
 
 (Optional) The fixed (i.e., non-overridable) value for the reserved parameter.
+If `fixed` is used, the `default` has no effect and should not be used.
```
Comment (wkande) [inline docs/ois/v1.0.0/ois.md]: The extra use of the word "**the**" before "**default**" throws me off, maybe:

If `fixed` is used, `default` has no effect and should not be used.

File: docs/ois/v1.0.0/ois.md [hunk 2]
```diff
@@ -457,11 +458,13 @@ Allowed values: `_type`, `_path` or `_times`
...
 
 #### 5.4.3. `default`
 
 (Optional) The default value for the reserved parameter. Used when no value is
-provided.
+provided. If `default` is used, the `fixed` must not be used because it would
```
Comment (wkande) [inline docs/ois/v1.0.0/ois.md]: Same here.

If `default` is used, `fixed` must not be used because it would...

**Review by dcroote — APPROVED** [review]
@Siegrift  - I added your fix to the previous `config.json` pages, hope that was OK

**Comment by Siegrift** [comment]
>  I added your fix to the previous config.json pages, hope that was OK

Totally, thanks for that.

**Review by wkande — APPROVED** [review]
I did one tiny grammar change. Will merge now.

**Pull request merged**